examples/llama/llama-loop.py

```diff
@@ -6,30 +6,34 @@
 tokenizer=model.create_tokenizer()
 print("Tokenizer created")
 
-prompt = "I like walking my cute dog"
+prompts = ["I like walking my cute dog",
+           "What is the best restaurant in town?",
+           "Hello, how are you today?"]
 
-input_tokens = tokenizer.encode(prompt)
+input_tokens = tokenizer.encode_batch(prompts)
 
-params=og.SearchParams(model)
+params=og.GeneratorParams(model)
 params.max_length = 256
-params.input_ids = input_tokens
+params.set_input_sequences(input_tokens)
+
+print("Generator created")
```

Comment:
Is the placement of this print statement incorrect?

Reply:
Fixed